src/main/java/frc/robot/commands/FancySwerveGyroReset.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
Is this file needed or used anywhere? It doesn't look like it contains anything useful, from my initial glance.